src/layout/output.cpp

```diff
@@ -19,12 +19,18 @@
 namespace chopper::layout
 {
 
-void write_user_bins_to(std::vector<std::string> const & filenames, std::ostream & stream)
+void write_user_bins_to(std::vector<std::vector<std::string>> const & filenames, std::ostream & stream)
 {
     stream << chopper::prefix::meta_chopper_user_bins_start << '\n';
     size_t counter{};
-    for (auto const & filename : filenames)
-        stream << seqan::hibf::prefix::meta_header << counter++ << ' ' << filename << '\n';
+    for (auto const & filenames_of_user_bin : filenames)
+    {
+        stream << seqan::hibf::prefix::meta_header << counter++ << ' ';
+        for (std::string const & filename : filenames_of_user_bin)
+            stream << filename << ' ';
```

Comment:
```suggestion
        stream << seqan::hibf::prefix::meta_header << counter++ << ' ';
        char separator = '';
        for (std::string const & filename : filenames_of_user_bin)
        {
            stream << separator << filename;
            separator = ',';
        } 
```
Something like that should work and then you don't need the `seekp`.